_includes/manuals/nightly/2.1_quickstart_installation.md

```diff
@@ -70,8 +70,9 @@ sudo yum -y install https://yum.theforeman.org/releases/{{page.version}}/el7/x86
 {% highlight bash %}
 sudo yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
If we make this a CentOS section, it can also be `sudo yum -y install epel-release` since it's in CentOS extras. You can even combine the two lines and `yum install epel-release centos-release-scl-rh`.